### DISCUSSION TITLE
vehicle

### DISCUSSION THREAD

**Discussion by Jayathu-SH** [Hirun4/MERN-HealthSlot#55] [Q&A] [answered]
what is your favourite vehicle

**Comment by sandarunihara** [comment, marked as answer]
ford mustang

**Comment by Ben-Henry** [comment]
I love my Ford truck it’s reliable and perfect for hauling materials, especially when working as a [Roofing Contractor in Hartsville](http://roofingcontractorhartsville.com). It handles everything from supplies to job site visits!
